js/ui/appDisplay.js

```diff
@@ -412,7 +412,7 @@ const AllViewContainer = new Lang.Class({
         this.gridActor = gridActor;
 
         gridActor.y_expand = true;
-        gridActor.y_align = Clutter.ActorAlign.CENTER;
+        gridActor.y_align = Clutter.ActorAlign.START;
```

Comment:
This, combined with the fact that we are allocating all the available height to the grid now will result in a non-centered grid on the screen, which will also push the search entry to the top, being more noticeable the more icons we have on screen. See the screenshots I pasted in the ticket.